src/routes.js

```diff
@@ -6,6 +6,7 @@ const faq = require('./api/faq/faq.routes');
 const admin = require('./api/admin/admin.routes');
 const history = require('./api/history/history.routes');
 const video = require('./api/video/video.routes');
+const SwaggerDocUI = require('./api/docs/doc.routes');
```

Comment:
Can you change this variable name to `doc` or `docs` to keep it more consistent?

Reply:
`documentation`? To keep it descriptive?

Reply:
Documentation works